src/components/tables/room-occupancy-table/OccupancyTable.js

```diff
@@ -4,10 +4,10 @@ import OccupancyTableHeading from "./OccupancyTableHeading";
 import OccupancyTableCell from "./OccupancyTableCell";
 import OccupancyTableRow from "./OccupancyTableRow";
 import OccupancyActionsTableCell from "./OccupancyActionsTableCell";
+import { TWO } from "../../../utils/constants";
```

Comment:
why this file is changed ?
please roll it back if doe not belongs to the fix

Reply:
This was an eslint requirement